### PR TITLE
add AggregateAboveBaseFee settings so lotus will aggregate more intuitively 

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/miner-configuration.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- It looks like there is a word missing here. Did you mean “listen to address”?
  ```
  ...tings of the miner API: As you see, the listen address is bound to the local...
                                             ^^^^^^^^^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ... noted in the setup instructions, it is very important to adjust this sectio...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...y benchmarking or by pledging a sector. WARNING The final value of ExpectedS...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...soon. # Publishing several deals in one message The PublishStorageDeals mess...
                                             ^^^^^^^
  ```
- Do not use a colon (:) before a series that is introduced by a preposition (‘as’). Remove the colon or add a noun or a noun phrase after the preposition.
  ```
  ...ompletion, its exit code is interpreted as: 0: success, proceed with the dea...
                                             ^^^
  ```
- Possibly missing article found.
  ```
  ...valFilter = "/bin/false". /bin/false is binary that immediately exits with a...
                                             ^^^^^^
  ```
- If this is a compound adjective that modifies the following noun, use a hyphen.
  ```
  ...art relatively soon. You can also use a third party content policy framework...
                                             ^^^^^^^^^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...# Sealing section This section controls some of the behavior around sector s...
                                             ^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...s hit: MaxPreCommitBatch is the maximum amount of sectors' pre-commitments t...
                                             ^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...et expires WILL cause the whole message fail. Therefore, we recommend miners...
                                             ^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...tions. MinPreCommitBatch is the minimum amount of sectors' pre-commitment to...
                                             ^^^^^^
  ```
- The word “therefore” is an adverb that can’t be used like a conjunction, and therefore needs to be separated from the sentence.
  ```
  ...orsBatch cost less gas than sending via PreCommitSector, therefore, we recom...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb ‘recommend’ is used with the gerund form.
  ```
  ...ding via PreCommitSector, therefore, we recommend to keep this value as 1. N...
                                             ^^^^^^^^^^^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...k is hit: MaxCommitBatch is the maximum amount of sectors' prove-commitments...
                                             ^^^^^^
  ```
- The word ‘prove’ is a verb. Did you mean the noun “proof”?
  ```
  ... base fee is lower than this value, the prove commits will be submitted indi...
                                             ^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ... in burn. MinCommitBatch is the minimum amount of sectors' prove-commitment ...
                                             ^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...al seal workers, you may want to modify some of the options. # Fees section ...
                                             ^^^^^^^^^^^
  ```
- Did you mean “setting”? Or maybe you should add a pronoun? In active voice, ‘allow’ + ‘to’ takes an object, usually a pronoun.
  ```
  ... # Fees section The fees section allows to set limits to the gas consumption...
                                             ^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ...se fee for the messages to be included. A very large max fee can, however, r...
                                             ^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...es when network fees are high. For more details see the Miner addresses sect...
                                             ^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/miner-configuration.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- It looks like there is a word missing here. Did you mean “listen to address”?
  ```
  ...tings of the miner API: As you see, the listen address is bound to the local...
                                             ^^^^^^^^^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ... noted in the setup instructions, it is very important to adjust this sectio...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...y benchmarking or by pledging a sector. WARNING The final value of ExpectedS...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...soon. # Publishing several deals in one message The PublishStorageDeals mess...
                                             ^^^^^^^
  ```
- Do not use a colon (:) before a series that is introduced by a preposition (‘as’). Remove the colon or add a noun or a noun phrase after the preposition.
  ```
  ...ompletion, its exit code is interpreted as: 0: success, proceed with the dea...
                                             ^^^
  ```
- Possibly missing article found.
  ```
  ...valFilter = "/bin/false". /bin/false is binary that immediately exits with a...
                                             ^^^^^^
  ```
- If this is a compound adjective that modifies the following noun, use a hyphen.
  ```
  ...art relatively soon. You can also use a third party content policy framework...
                                             ^^^^^^^^^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...# Sealing section This section controls some of the behavior around sector s...
                                             ^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...s hit: MaxPreCommitBatch is the maximum amount of sectors' pre-commitments t...
                                             ^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...et expires WILL cause the whole message fail. Therefore, we recommend miners...
                                             ^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...tions. MinPreCommitBatch is the minimum amount of sectors' pre-commitment to...
                                             ^^^^^^
  ```
- The word “therefore” is an adverb that can’t be used like a conjunction, and therefore needs to be separated from the sentence.
  ```
  ...orsBatch cost less gas than sending via PreCommitSector, therefore, we recom...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The verb ‘recommend’ is used with the gerund form.
  ```
  ...ding via PreCommitSector, therefore, we recommend to keep this value as 1. N...
                                             ^^^^^^^^^^^^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...k is hit: MaxCommitBatch is the maximum amount of sectors' prove-commitments...
                                             ^^^^^^
  ```
- The word ‘prove’ is a verb. Did you mean the noun “proof”?
  ```
  ... base fee is lower than this value, the prove commits will be submitted indi...
                                             ^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ... in burn. MinCommitBatch is the minimum amount of sectors' prove-commitment ...
                                             ^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...al seal workers, you may want to modify some of the options. # Fees section ...
                                             ^^^^^^^^^^^
  ```
- Did you mean “setting”? Or maybe you should add a pronoun? In active voice, ‘allow’ + ‘to’ takes an object, usually a pronoun.
  ```
  ... # Fees section The fees section allows to set limits to the gas consumption...
                                             ^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ...se fee for the messages to be included. A very large max fee can, however, r...
                                             ^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...es when network fees are high. For more details see the Miner addresses sect...
                                             ^^^^^^^
  ```

**Review by magik6k — APPROVED** [review]
Still have mentions of `MinPreCommitBatch` at ~L239, but looks good otherwise

**Comment by jennijuju** [comment]
> Still have mentions of `MinPreCommitBatch` at ~L239, but looks good otherwise

ah - only removed the configuration. fixed now, thanks

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/miner-configuration.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- It looks like there is a word missing here. Did you mean “listen to address”?
  ```
  ...tings of the miner API: As you see, the listen address is bound to the local...
                                             ^^^^^^^^^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ... noted in the setup instructions, it is very important to adjust this sectio...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...y benchmarking or by pledging a sector. WARNING The final value of ExpectedS...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...soon. # Publishing several deals in one message The PublishStorageDeals mess...
                                             ^^^^^^^
  ```
- Do not use a colon (:) before a series that is introduced by a preposition (‘as’). Remove the colon or add a noun or a noun phrase after the preposition.
  ```
  ...ompletion, its exit code is interpreted as: 0: success, proceed with the dea...
                                             ^^^
  ```
- Possibly missing article found.
  ```
  ...valFilter = "/bin/false". /bin/false is binary that immediately exits with a...
                                             ^^^^^^
  ```
- If this is a compound adjective that modifies the following noun, use a hyphen.
  ```
  ...art relatively soon. You can also use a third party content policy framework...
                                             ^^^^^^^^^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...# Sealing section This section controls some of the behavior around sector s...
                                             ^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...s hit: MaxPreCommitBatch is the maximum amount of sectors' pre-commitments t...
                                             ^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ...et expires WILL cause the whole message fail. Therefore, we recommend miners...
                                             ^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...k is hit: MaxCommitBatch is the maximum amount of sectors' prove-commitments...
                                             ^^^^^^
  ```
- The word ‘prove’ is a verb. Did you mean the noun “proof”?
  ```
  ... base fee is lower than this value, the prove commits will be submitted indi...
                                             ^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ... in burn. MinCommitBatch is the minimum amount of sectors' prove-commitment ...
                                             ^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...al seal workers, you may want to modify some of the options. # Fees section ...
                                             ^^^^^^^^^^^
  ```
- Did you mean “setting”? Or maybe you should add a pronoun? In active voice, ‘allow’ + ‘to’ takes an object, usually a pronoun.
  ```
  ... # Fees section The fees section allows to set limits to the gas consumption...
                                             ^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ...se fee for the messages to be included. A very large max fee can, however, r...
                                             ^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...es when network fees are high. For more details see the Miner addresses sect...
                                             ^^^^^^^
  ```

File: docs/mine/lotus/miner-configuration.md [hunk 1]
```diff
@@ -233,10 +233,10 @@ This section controls some of the behavior around sector sealing:
 
 In lotus v1.10.0 and up, if `BatchPreCommit` is set to false, pre-commitments will be sent to the chain via `PreCommitSector` messages once they are ready. If `BatchPreCommit` is set to true, lotus will batch pre-commitments until any of `MaxPreCommitBatch`, `PreCommitBatchWait` or `PreCommitBatchSlack` is hit:
 - `MaxPreCommitBatch` is the maximum amount of sectors' pre-commitments to batch in one `PreCommitSectorsBatch` message. According to FIP-0008, this values is up to 256.
-- `PreCommitBatchWait` is how long to wait before submitting the current batch **after** crossing `MinPreCommitBatch`. Note: the ticket of pre-commitment has an expiration of approximately **31.5 hours**, one sector's pore-commit ticket expires **WILL** cause the whole message fail. Therefore, **we recommend miners to set this value lower than 30 hours.**
+- `PreCommitBatchWait` is how long to wait before submitting the current batch. Note: the ticket of pre-commitment has an expiration of approximately **31.5 hours**, one sector's pore-commit ticket expires **WILL** cause the whole message fail. Therefore, **we recommend miners to set this value lower than 30 hours.**
```
Comment (arajasek) [inline docs/mine/lotus/miner-configuration.md:236]:
```suggestion
- `PreCommitBatchWait` is how long to wait before submitting the current batch. Note: the ticket of pre-commitment has an expiration of approximately **31.5 hours**, one sector's pre-commit ticket expires **WILL** cause the whole message to fail. Therefore, **we recommend miners to set this value lower than 30 hours.**
```

**Review by arajasek — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/mine/lotus/miner-configuration.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://networks.filecoin.io
- It looks like there is a word missing here. Did you mean “listen to address”?
  ```
  ...tings of the miner API: As you see, the listen address is bound to the local...
                                             ^^^^^^^^^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ... noted in the setup instructions, it is very important to adjust this sectio...
                                             ^^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...y benchmarking or by pledging a sector. WARNING The final value of ExpectedS...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...soon. # Publishing several deals in one message The PublishStorageDeals mess...
                                             ^^^^^^^
  ```
- Do not use a colon (:) before a series that is introduced by a preposition (‘as’). Remove the colon or add a noun or a noun phrase after the preposition.
  ```
  ...ompletion, its exit code is interpreted as: 0: success, proceed with the dea...
                                             ^^^
  ```
- Possibly missing article found.
  ```
  ...valFilter = "/bin/false". /bin/false is binary that immediately exits with a...
                                             ^^^^^^
  ```
- If this is a compound adjective that modifies the following noun, use a hyphen.
  ```
  ...art relatively soon. You can also use a third party content policy framework...
                                             ^^^^^^^^^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...# Sealing section This section controls some of the behavior around sector s...
                                             ^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...s hit: MaxPreCommitBatch is the maximum amount of sectors' pre-commitments t...
                                             ^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...k is hit: MaxCommitBatch is the maximum amount of sectors' prove-commitments...
                                             ^^^^^^
  ```
- The word ‘prove’ is a verb. Did you mean the noun “proof”?
  ```
  ... base fee is lower than this value, the prove commits will be submitted indi...
                                             ^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ... in burn. MinCommitBatch is the minimum amount of sectors' prove-commitment ...
                                             ^^^^^^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...ng queue, run: the output is the sector ids: To ignore the configuration and...
                                             ^^^
  ```
- If the text is a generality, ‘of the’ is not necessary.
  ```
  ...al seal workers, you may want to modify some of the options. # Fees section ...
                                             ^^^^^^^^^^^
  ```
- Did you mean “setting”? Or maybe you should add a pronoun? In active voice, ‘allow’ + ‘to’ takes an object, usually a pronoun.
  ```
  ... # Fees section The fees section allows to set limits to the gas consumption...
                                             ^^^^^^
  ```
- As an alternative to the over-used intensifier ‘very’, consider replacing this phrase.
  ```
  ...se fee for the messages to be included. A very large max fee can, however, r...
                                             ^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...es when network fees are high. For more details see the Miner addresses sect...
                                             ^^^^^^^
  ```

**Pull request merged**